src/Pages/Good_to_know.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function Good_to_know() {
```

Comment:
Use Camel case to keep consistent

Reply:
> Use Camel case to keep consistent → GoodToKnow
Done! Please check if anything else is needed!